scripts/clean_osm_data.py

```diff
@@ -835,6 +835,53 @@ def set_name_by_closestcity(df_all_generators, colname="name"):
     return df_all_generators
 
 
+def load_network_data(network_asset):
```

Comment:
Fantastic!!! :D
This function is what we need :)
As little comments: please, add an argument to the dunction like "clean_osm_data_options".
We try to avoid using global values within functions such as "snakemake" as this creates nested dependencies that are hard to maintain.
Moreover, please reduce the number of empty lines: no empty line between a comment and the code for example.
Moreover, no empty line after the docstring ending with """